packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -110,11 +110,20 @@ const bootstrapNewUser = async (
           type: 'inviteYourTeam' as const,
           priority: 2
         }
+        // We're racing with accept team invitation here which also adds some suggested actions. We don't want this to fail just because of duplicates.
```

Comment:
While this is true, the more common case where the bug occurs is bootstrap without autojoin which adds the `createNewTeam` suggested action which then is also added in `acceptTeamInvitation`.